src/main/resources/liquibase/crop_changelog/19_4_3.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
@mderamos Not so sore why this is commited to this branch? Could you please check? 
CC @cuenyad